vk_video_decoder/libs/VkVideoParser/VulkanVideoParser.cpp

```diff
@@ -1544,6 +1653,8 @@ uint32_t VulkanVideoParser::FillDpbH265State(
         pStdPictureInfo->RefPicSetLtCurr[numPocLtCurr++] = 0xff;
     }
 
+    (void)numPocTotalCurr; // TODO
```

Comment:
TODO what and why?

Reply:
This variable is unused, it needs to be removed or used.